server/services/apollo/src/index.js

```diff
@@ -170,4 +183,28 @@ async function runServerForever() {
   }
 }
 
+async function send_telemetry_event(event) {
+  if (TELEMETRY_ENABLED) {
+    try {
+      // TODO add timeout
+      fetch('https://sens-o-matic.prefect.io/', {
+        method: 'post',
+        body: JSON.stringify({
+          source: 'prefect_server',
+          type: event,
+          payload: { id: TELEMETRY_ID }
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+          'X-Prefect-Event': 'prefect_server-0.0.1'
+        }
+      })
+        .then(res => res.json())
+        .then(json => log(json))
```

Comment:
Do we want to log these responses, or is this leftover from debugging? If so, we should probably add some context around the log message.

Reply:
Since these messages are sent pretty infrequently I think it makes sense to log them. Thoughts @cicdw @jlowin ?

Context message could be: "Telemetry sent to Prefect Technologies, Inc:" or something similar?

Reply:
> Context message could be: "Telemetry sent to Prefect Technologies, Inc:" or something similar?

Perhaps log what we send (not what's received, as is done now) along with a message like that?

Reply:
Yea logging them for now makes sense (since they're currently only on spin up / tear down).  And yea I like that context message 👍

Reply:
> since they're currently only on spin up / tear down

They're sent every 10 minutes in the current implementation as a heartbeat.

Reply:
Ah sorry I missed that; I think that's minimal enough for now but there's a risk of generating excessive noise if / when we send more frequently than that.